a10_octavia/tests/unit/controller/worker/tasks/test_virtual_port_tasks.py

```diff
@@ -32,6 +32,7 @@
 
 VTHUNDER = data_models.VThunder()
 LB = o_data_models.LoadBalancer(id=a10constants.MOCK_LOAD_BALANCER_ID)
+update_dict = {}
```

Comment:
Since its a global variable, should be capitalized

Reply:
Capitalized